resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -145,14 +145,10 @@
         <executions>
           <execution>
             <id>test</id>
+            <phase>none</phase>
             <goals>
               <goal>test</goal>
             </goals>
-            <configuration>
```

Comment:
Do we not need this anymore because the integration test is hidden behind the profile?

Reply:
We don't need this because I disabled the execution the proper way a few lines above.